include/IndexStoreDB/Index/IndexSystem.h

```diff
@@ -47,6 +47,7 @@ class INDEXSTOREDB_EXPORT IndexSystem {
                                              std::shared_ptr<IndexStoreLibraryProvider> storeLibProvider,
                                              std::shared_ptr<IndexSystemDelegate> Delegate,
                                              bool useExplicitOutputUnits,
+                                             bool disableOutOfDateFileTriggers,
```

Comment:
This option should align with `listenToUnitEvents`, since they are both related to file watching. I suggest putting these arguments next to each other in the list and rewording it so the new option is positive instead of negative.  I'm less concerned about the specific name, but it should enable something not disable it.